docs/pages/jpscore/jpscore_trains.md

```diff
@@ -5,189 +5,133 @@ tags: [jpscore, file, simulation]
 sidebar: jupedsim_sidebar
 folder: jpscore
 permalink: jpscore_trains.html
-summary: The interaction with trains is modeled with an event-based deleting and creating of doors. When a train arrives on the platform, doors are created and again deleting when the train departs.
-last_updated: Dec 31, 2019
+summary: The interaction with trains is modeled with an event-based deleting and creating of doors. When a train arrives on the platform, doors are created and again deleted when the train departs.
+last_updated: Jul 03, 2020
 ---
 
 
 ## Train constraints
-
 Information regarding trains are organized in two different files:
 - [Train timetable](#train-timetable): This file defines arrival and departure times of trains.
```

Comment:
- [Train timetable](#train-timetable): This file defines arrival and departure times of trains and specifies their location on tracks.

Reply:
Done